rosdep/python.yaml

```diff
@@ -2811,6 +2811,10 @@ python-plyfile-pip:
 python-poster:
   debian: [python-poster]
   ubuntu: [python-poster]
+python-pqdict:
```

Comment:
For a pip package, we require a `-pip` suffix for the key.

Reply:
Done.